### DISCUSSION TITLE
Change SDDM theme

### DISCUSSION THREAD

**Discussion by grdsdr58** [lxqt/lxqt#2621] [General]
Hi,
I want to change SDDM theme.
I tested archlinux theme with "$ sddm-greeter-qt6 --test-mode --theme /usr/share/sddm/themes/archlinux" and it is OK.
I changed in the file /lib/sddm/sddm.conf.d/default.conf "current=archlinux", but after restart SDDM does not start.
Why?

**Comment by Bluey26** [comment]
~Hi, try changing `/etc/sddm.conf`~

~~```
~~[Theme]
~~Current=archlinux
~~```

~~make a backup of the `sddm.conf ` file before doing any changes.~~

You can always use sddm-conf (https://github.com/qtilities/sddm-conf/issues) to change sddm options.

Regarding sddm not starting, i would suggest you to re-generate the file  `/lib/sddm/sddm.conf.d/default.conf`, maybe there was something wrong and now it does not work because of that.

**Update:** After reading the wiki page, i realized that the /etc/sddm.conf file was generated by `sddm-conf`(https://wiki.archlinux.org/title/SDDM#Configuration)

Regarding you issue, i just have found this:

>**Is it worth adding a dependency for qt5-declarative?**

>That is currently an optional dep of sddm and a lot of people are installing this theme and **getting a black screen**. The solution is simple, just to install that package, so I understand if you don't want to, especially with sddm saying you need it for qt5 themes, but people seem to be missing it.

>So I'd like to make the suggestion to include it as a dependency of this theme package
(from: https://aur.archlinux.org/packages/archlinux-themes-sddm)

Maybe that is needed for that package to work. My custom theme was just a copy/paste from default theme with background changed, i never have experienced this.

**Reply by grdsdr58** [reply]
Thanks Bluey26,
adding qt5-declarative worked.
